src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Areas/Identity/Pages/Account/ExternalLoginTests.cs

```diff
@@ -329,21 +346,6 @@ await using (var contentDbContext = InMemoryApplicationDbContext(contextId))
                     .AsQueryable()
                     .SingleAsync(invite => invite.Email == email);
                 Assert.True(newUsersInvite.Accepted);
-                
```

Comment:
Why remove this check out of interest?  It's here to make sure that the processed invites are limited to those of the currently logging-in user.

Reply:
Good spot, I think I intended to delete a block below it and made a mistake. I've reinstated it now.

I've also added some new checks to make sure the release role invites and publication role invites are left alone.